environment_setup.sh

```diff
@@ -73,7 +73,7 @@ if confirmupdate "Would you like to install local development programs like PHPS
   # Set some standard git configuration
   git config --global pull.rebase true
 
-  export PATH=./vendor/bin:~/.composer/vendor/bin:/usr/local/bin:/usr/local/sbin:$PATH
+  export PATH=./vendor/bin:~/.composer/vendor/bin:$PATH
```

Comment:
These no longer exist for me so assuming we don't need them anymore

Reply:
@olivierbouwman - I believe we still need `/opt/homebrew/bin` in our path - do you see brew binaries there on your machine?